src/components/forums/forum/ForumContent.tsx

```diff
@@ -346,20 +457,75 @@ export function ForumContent(props: ForumContentProps) {
+                    </div>
+                    <select 
+                      value={tokenGateSelection}
+                      onChange={(e) => setTokenGateSelection(e.target.value)}
+                      >
+                      <option value="">Select a token type</option>
+                      <option value="NFT">Metaplex NFT</option>
+                      {(!isSuccess(forumData.restriction)) && <option value="SPL">SPL Token</option>}
+                    </select>
+                  </div>
+                  {addNFTGate && (
+                      <div>
+                    <label className="addModeratorsLabel">
+                      Add new NFT Collection ID
+                    </label>
+                    <div>
+                      You can enter a list of Metaplex NFT Collection IDs here such that only
+                      holders of NFTs in the collection can participate in this
+                      forum.
+                      </div>
+                    <input
+                      type="text"
+                      placeholder="Enter NFT Collection IDs as a comma separted list"
+                      className="newAccessToken"
+                      name="accessToken"
+                      value={newForumAccessToken}
+                      onChange={(e) => setNewForumAccessToken(e.target.value)}
+                      />
+                    </div>)}
+                      {addSPLGate && (
+                            <div className="addSPLToken">
+                            <span className="createTopicLabel">
+                              Limit post access by SPL Collection ID
+                            </span>
+                            <input
+                              type="text"
+                              placeholder="Token mint ID"
+                              className="newAccessToken"
+                              name="accessToken"
+                              value={newForumAccessToken}
+                              onChange={(e) => {
+                                setNewForumAccessToken(e.target.value);
+                              }
+                              }
+                            />
+                            <span className="createTopicLabel">
+                              Amount
+                            </span>
+                            <Input
+                              type='number'
+                              value={1.00}
+                              min={0}
+                              step={0.01}
+                              className="newAccessToken"
+                              onChange={(value) => {
+                                setNewForumAccessTokenAmount(parseFloat(value));
+                              }
+                              }/>
+                          </div>
+                            )}
+                  {newForumAccessToken.length > 0 && (
+                    accessList.map((pubkey) => (
```

Comment:
when multiple tokens are added, style the list as it is on the `createForum` component

![image](https://user-images.githubusercontent.com/47254484/192607115-c758389a-b1c6-4feb-9c43-5c285405a555.png)